### PR TITLE
Fixing button text in start trial modal

### PR DESCRIPTION
**Pull request by ctlaltdieliet**
#### Summary
The text at start_trial.modal_btn.start isn't the same as mentioned in start_trial.modal.disclaimer
A translator noticed this
#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/20605
#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by AGMETEOR — APPROVED** [review]

File: i18n/en.json [hunk 1]
```diff
@@ -4586,7 +4586,7 @@
   "start_trial.modal_btn.start": "Start free 30-day trial",
   "start_trial.modal_btn.start_free_trial": "Start free 30-day trial",
   "start_trial.modal_title": "Start your free Enterprise trial now",
-  "start_trial.modal.disclaimer": "By clicking “Start 30-day trial”, I agree to the <linkEvaluation>Mattermost Software Evaluation Agreement</linkEvaluation>, <linkPrivacy>privacy policy</linkPrivacy> and receiving product emails.",
+  "start_trial.modal.disclaimer": "By clicking “Start 30-day free trial”, I agree to the <linkEvaluation>Mattermost Software Evaluation Agreement</linkEvaluation>, <linkPrivacy>privacy policy</linkPrivacy> and receiving product emails.",
```
Comment (jgilliam17) [inline i18n/en.json:4589]:
```suggestion
  "start_trial.modal.disclaimer": "By clicking “Start free 30-day trial”, I agree to the <linkEvaluation>Mattermost Software Evaluation Agreement</linkEvaluation>, <linkPrivacy>privacy policy</linkPrivacy> and receiving product emails.",
```
I think `free` should be placed before `30-day` . Maybe @cwarnermm can confirm - thanks 🙂

**Review by jgilliam17 — CHANGES_REQUESTED** [review]
Thanks @ctlaltdieliet 
one word placement issue
Screen Shot 2022-07-15 at 7 19 37 PM

**Review by jgilliam17 — APPROVED** [review]
Thanks @ctlaltdieliet 
Tested, looks good to merge.
Verified updated button text.
[E2E report ](https://automation-dashboard.vercel.app/cycles/45f68b75-c0f3-4904-a8ed-838915768f8c)- no PR related failures.

**Comment by jgilliam17** [comment]
@devinbinnie Do you mind quickly checking and approving the last workflow so we can merge? Thanks 🙂

**Pull request merged**